lib/RoadizRozierBundle/config/services.yaml

```diff
@@ -6,6 +6,7 @@ services:
         autoconfigure: true # Automatically registers your services as commands, event subscribers, etc.
         bind:
             $backofficeMenuEntries: '%roadiz_rozier.backoffice_menu_configuration%'
+            $bookmarksConfiguration: '%roadiz_rozier.bookmarks_configuration%'
```

Comment:
You do not need to bind `$bookmarksConfiguration` here, you only use  `%roadiz_rozier.bookmarks_configuration%` in the `RZ\Roadiz\RozierBundle\Model\BookmarkCollection` factory arguments.